game/scripts/vscripts/components/duels/duels.lua

```diff
@@ -177,12 +177,18 @@ function Duels:CheckDuelStatus (hero)
   end
 
   if not player.assigned or not player.duelNumber then
+    DebugPrint('Player died who isnt in a duel?')
```

Comment:
I think we should make this an `error`.

Reply:
i'd rather have it logged that it happens then see the behavior after it than to hard fail out, error keeps the rest of the code from running